ci/.gitignore

```diff
@@ -0,0 +1 @@
+/bazel-*
```

Comment:
i think think this is not required ... `.gitignore` is kept at repo root and already contains this exclusion